spec/classes/corosync_spec.rb

```diff
@@ -442,6 +442,28 @@
       end
     end
 
+    context 'with undefined watchdog_device' do
+      it 'does not configure watchdog_device resource' do
+        is_expected.to contain_file('/etc/corosync/corosync.conf').without_content(
+          %r{watchdog_device:}
+        )
+      end
+    end
+
+    context 'with defined watchdog_device' do
+      before do
+        params.merge!(
+          watchdog_device: 'off'
+        )
+      end
```

Comment:
This is a bad practice and it's better to override it like this:
```suggestion
      let(:params) { super().merge(watchdog_device: 'off') }
```